intel_extension_for_transformers/backends/neural_engine/graph/README.md

```diff
@@ -65,11 +65,11 @@ Running LLAMA model, for details please refer to [LLaMA model documentation](./a
 OMP_NUM_THREADS=56 numactl -m 0 -C 0-55 ./build/bin/main_llama -m ~/llama.cpp/models/ne-model-q4_j.bin --seed 12 -c 512 -b 1024 -n 256 --keep 48 -t 56 --repeat-penalty 1.0 --color -p "She opened the door and see"
 ```
 
-Running GPT-NEOX / MPT / FALCON / / GPT-J / STARCODER model, please use `main_gptneox` / `main_mpt` / `main_falcon` / `main_starcoder` (Please type **prompt about codes** when use `STARCODER`. For example, `-p "def fibonnaci("`).
+Running GPT-NEOX / MPT / FALCON / / GPT-J / STARCODER model, please use `chat_gptneox` / `chat_mpt` / `chat_falcon` / `chat_starcoder` (Please type **prompt about codes** when use `STARCODER`. For example, `-p "def fibonnaci("`).
```

Comment:
why chat? is llm more neutral?

Reply:
The prefix `chat_` is for the chat application for now. There will be more applications in the future. Do I understand it correctly @airMeng ?